src/renderer/address-book/icon/index.jsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
```

Comment:
@eduardosmoura forgot to tell you but we leaving the icons on selfkey-ui so it can be reused. So you need to export it here https://github.com/SelfKeyFoundation/selfkey-ui/blob/master/src/index.tsx#L96 to be able to import in the wallet. Cheers